public_html/wp-content/plugins/multi-event-sponsors/package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "multi-event-sponsors",
+	"version": "0.1.0",
+	"license": "GPL-2.0-or-later",
+	"eslintConfig": {
+		"extends": "../../../../.eslintrc.js"
+	},
+	"scripts": {
+		"build": "wp-scripts build javascript/index.js --output-path=build",
+		"start": "wp-scripts start javascript/index.js --output-path=build --hot",
+		"lint:js": "wp-scripts lint-js",
+		"packages-update": "wp-scripts packages-update"
+	},
+	"devDependencies": {
+		"@wordpress/icons": "^7.0.1",
+		"@wordpress/scripts": "^22.0.1"
```

Comment:
Oh, I also realized that you'll need to add a `lint:css` script once you add the package to the workspaces list (the [github action script requires it](https://github.com/WordPress/wordcamp.org/blob/6821bdcaf68e9d22292f5f510fe8645849412646/.github/workflows/linter.yml#L46)). When I tried it, I got an error because this version of wp-scripts doesn't match the rest, I guess we need to update the packages. For now, you can pin this to `"@wordpress/scripts": "19.2.3"` and use `"lint:css": "wp-scripts lint-style css"`.

Reply:
okay, but you'll still need to add a lint:css script, otherwise the lint action will fail. `"lint:css": "echo \"No linter\""` or something should work